### PR TITLE
fix(code-block): remove extra z-index [khcp-5683]

### PR DESCRIPTION
**Pull request by aanchalm01**
# Summary
Addresses:
[https://konghq.atlassian.net/browse/KHCP-5683](https://konghq.atlassian.net/browse/KHCP-5683)

Before:
image 839

After:
image 840
## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/components/KCodeBlock/KCodeBlock.vue [hunk 1]
```diff
@@ -1011,7 +1011,6 @@ $dark-focusColor: var(--green-500, color(green-500));
   position: absolute;
   top: var(--spacing-xs, spacing(xs));
   right: var(--spacing-md, spacing(md));
-  z-index: 2;
```
Comment (adamdehaven) [inline src/components/KCodeBlock/KCodeBlock.vue:1014]: I don't think this should be removed; instead the `.k-modal-fullscreen-header` likely just needs a higher `z-index`

Does adding `z-index` there cause issues?

**Review by adamdehaven — CHANGES_REQUESTED** [review]

File: src/components/KCodeBlock/KCodeBlock.vue (hunk 1 above)
Comment (aanchalm01) [reply src/components/KCodeBlock/KCodeBlock.vue:1014]: What's the reasoning behind `z-index: 2` here? I checked the stack order and the tab sequence and its same without it.

Comment (adamdehaven) [reply src/components/KCodeBlock/KCodeBlock.vue:1014]: TBH I'm not 100%. @kleinfreund created the component; I'm guessing it's to keep the copy button over the code.

What if we just drop it to `1` instead? I still think we should bump the z-index of the modal header

Comment (aanchalm01) [reply src/components/KCodeBlock/KCodeBlock.vue:1014]: Updated.

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**